packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/NftForm.tsx

```diff
@@ -79,22 +77,37 @@ export const NftForm: React.FC<NftFormProps> = ({ setFormStatus, onSubmit, video
       }),
       z.object({
         type: z.literal('duration'),
-        durationDays: z.number().nullable(),
+        durationDays: z.number(),
       }),
     ])
     .nullable()
     .optional()
 
   const schema = z.object({
```

Comment:
Could we maybe move this to a function that creates this schema given necessary values from the form? It's crowding the component a bit